evalml/automl/automl_algorithm/automl_algorithm.py

```diff
@@ -80,3 +84,48 @@ def pipeline_number(self):
     def batch_number(self):
         """Returns the number of batches which have been recommended so far."""
         return self._batch_number
+
+    def _create_ensemble(self):
+        next_batch = []
+
+        # Custom Stacked Pipelines
+        problem_type = list(self._best_pipeline_info.values())[0][
+            "pipeline"
+        ].problem_type
```

Comment:
This is a hard-to-read way of fetching the problem type, and the long call of `list(self._best_pipeline_info.values())` is used more than once - it might be worth it to factor that bit out into its own descriptively-named variable!